gradle/script-git-version.gradle

```diff
@@ -11,7 +11,7 @@ import org.ajoberstar.grgit.Grgit
 
 ext {
     git = Grgit.open(currentDir: projectDir)
-    def tagSize = git.tag.list().findAll { (it.name =~ "android-v*" || it.name =~ "v1\\d.*") }.size()
+    def tagSize = git.tag.list().findAll { (it.name =~ "android-v*" || it.name =~ "v1\\d.*") }.size() + 1
```

Comment:
why did you add `+1`?

Reply:
we provided an extra build to rave team manually. this was added to remove the error regarding versionCode should be greater than older versionCode in playstore

Reply:
ah. got it
maybe you should add extra comment here